apps/frontend/public/static/export/template.html

```diff
@@ -0,0 +1,294 @@
+            {{#files}}
+            <div class="col-xl results-set">
+              <span>Filename: <strong>{{filename}}</strong></span><br />
+              <span>Tool Version: <strong>{{toolVersion}}</strong></span><br />
+              <span>Platform: <strong>{{platform}}</strong></span><br />
+              <span>Duration: <strong>{{duration}}</strong></span><br />
+            </div>
+            {{/files}}
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="container-fluid pt-4" id="statusCards">
+      <div class="row">
+        <div class="col-md">
+          <div class="card bg-success tall-card">
+            <div class="card-body">
+              <div class="card-title">
+                <div class="h4">
+                  <svg style="width:24px; height:24px" viewBox="0 0 24 24">
+                    <path fill="white" d="{{icons.circleCheck}}" />
+                  </svg>
+                  <span id="passedCount">Passed: {{statistics.passed}}</span>
+                </div>
+                <p id="passedCountSubtitle" class="card-text"> {{statistics.passedTests}} individual checks passed. </p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-md">
+          <div class="card bg-danger">
+            <div class="card-body tall-card">
+              <div class="card-title">
+                <div class="h4">
+                  <svg style="width:24px;height:24px" viewBox="0 0 24 24">
+                    <path fill="white" d="{{icons.circleCross}}" />
+                  </svg>
+                  <span id="failedCount">Failed: {{statistics.failed}}</span>
+                </div>
+                <p class="card-text" id="failedCountSubtitle"> {{passingTestsFailedControl}} individual checks passed, {{failedTests}} failed out of {{totalTests}} total checks </p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-md">
+          <div class="card bg-info tall-card">
+            <div class="card-body">
+              <div class="card-title">
+                <div class="h4">
+                  <svg style="width:24px;height:24px" viewBox="0 0 24 24">
+                    <path fill="white" d="{{icons.circleMinus}}" />
+                  </svg>
+                  <span id="naCount">Not Applicable: {{statistics.notApplicable}}</span>
+                </div>
+                <p class="card-text" id="naCountSubtitle"> System exception or absent component </p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-md">
+          <div class="card bg-warning tall-card">
+            <div class="card-body">
+              <div class="card-title">
+                <div class="h4">
+                  <svg style="width:24px;height:24px" viewBox="0 0 24 24">
+                    <path fill="white" d="{{icons.circleAlert}}" />
+                  </svg>
```

Comment:
If the whole webfont is available as part of the page that may simplify actually adding the icons to this page. It would be nice if this was something similar to `<i class="{{icons.circleAlert}}" />`. If that isn't possible with the current state, it looks like the CSS may make something like `<i class="mdi-icon-name" />` possible.